plugins/dev-create/bin/create.js

```diff
@@ -59,7 +59,7 @@ const pluginType = program.type || 'plugin';
 const pluginDir = program.dir ||
     (pluginType == 'plugin' ? pluginName : `${pluginType}-${pluginName}`);
 const pluginPath = path.join(root, pluginDir);
-const pluginAuthor = program.author || 'Blockly Team';
+const pluginAuthor = program.author || '';
```

Comment:
Could we have this still be Blockly team if it is a first party plugin?

Reply:
Good point. Yes, will fix.

Reply:
done.